### ISSUE TITLE
`triton volume ls -l` should output a `RESOURCE` column

### ISSUE THREAD

**Issue by misterdjules** [joyent/node-triton#217]
When listing NFS volumes, the remote filesystem path is not outputted, including when using the `-l` command line option.

Even if it's not considered to be the main use case currently, users who `mount` such volumes manually (i.e not via docker or CloudAPI's CreateMachine endpoint) need to get that remote path. 

Currently, they need to run another `triton volume get` command. It would be handy for them to have that info at hand in the `volume list` output too.

FWIW, it would also make the implementation consistent with [what the current version of the RFD documents for `triton volume list`](https://github.com/joyent/rfd/blob/05f9f55a40877fb77c65ce004c4045ca45f76b14/rfd/0026/README.md#list).

**Comment by misterdjules** [comment]
CR at https://cr.joyent.us/#/c/2296/. That CR doesn't contain any test so far, which I find annoying. I'm just wondering if such tests are worth implementing, since it seems the current code base doesn't test the form the output of at least some commands.

This is how `triton volume ls -l`'s output looks like with that change:

1. When volume `fd54ffd8` is being created:

    ```
     ➜  node-triton git:(master) ✗ ./bin/triton volumes                                           
    SHORTID   NAME                           SIZE   TYPE       RESOURCE                      STATE     AGE
    5143368c  test-vol-from-node-triton      10240  tritonnfs  192.168.128.10:/exports/data ready     10m
    fd54ffd8  test-vol-from-node-triton-bis  10240  tritonnfs  -                             creating  5s
    ➜  node-triton git:(master) ✗
    ```
2. When volume `fd54ffd8` is created:

    ```
    ➜  node-triton git:(master) ✗ ./bin/triton volumes
    SHORTID   NAME                           SIZE   TYPE       STATE  AGE
    5143368c  test-vol-from-node-triton      10240  tritonnfs  ready  23m
    fd54ffd8  test-vol-from-node-triton-bis  10240  tritonnfs  ready  12m
    ➜  node-triton git:(master) ✗ 
    ```

**Comment by trentm** [comment]
Re testing: Julien and I discussed in meatspace. My opinion: I don't want the *tabular* output of `triton` commands to be a promised interface. The goal for tabular output is to be useful for a human reading it. The promised interfaces are the methods to output data for code to parse. Namely: using `-H -o FIELDS` or using `-j` for JSON output of the raw fields from CloudAPI.

**Comment by rmustacc** [comment]
I would actually go one step further and unless someone has -j or an equivalent for non-json like -p for pasreable that the output can always change. That way we can use nicenum for things like SIZE, etc. We've found that basically saying only the parseable output with the additional requirement of specifying things like -H -o FIELDS have been really useful in the OS.

**Comment by trentm** [comment]
@rmustacc `triton` *does* have cases (currently just the one I think) of "nicenum"-like usage:

```
$ triton pkgs
SHORTID   NAME             MEMORY  SWAP  DISK  VCPUS
14ad9d54  g4-highcpu-128M    128M  512M    3G      -
14ae2634  g4-highcpu-256M    256M    1G    5G      -
14aea8fc  g4-highcpu-512M    512M    2G   10G      -
```

and a `-p` for that case:

```
$ triton pkgs -p
SHORTID   NAME             MEMORY    SWAP     DISK  VCPUS
14ad9d54  g4-highcpu-128M     128     512     3072      0
14ae2634  g4-highcpu-256M     256    1024     5120      0
14aea8fc  g4-highcpu-512M     512    2048    10240      0
```

Do you think it sufficient that the promise is "if there is a '-p' option, then you have to use it for stable output" or should we be lookign at having (possible no-op) `-p` options on all commands that output tabular data?

**Comment by rmustacc** [comment]
I dunno. Maybe I'd say unless there's a -p/-j then there's no promise?

**Comment by misterdjules** [comment]
Merged with c56d0a25da0f70ed9c2f91ce2f6f361cb6ef5f29.
